.gitignore

```diff
@@ -7,6 +7,8 @@ coverage
 .output
 .env
 dist
+dist-ssr
+components.d.ts
```

Comment:
The official stance on whether to add `components.d.ts` to source control is mixed, with pros and cons on both sides, but since this project did not already have it, I erred on the don't-add-it side.